### ISSUE TITLE
Why maximum dependency length is set to 100 in DAG?

### ISSUE THREAD

**Issue by egorzekov** [jaegertracing/jaeger-ui#130]
https://github.com/jaegertracing/jaeger-ui/blob/85f9822b20a261950117c53ae120552e251d8f55/src/components/DependencyGraph/index.js#L88

Hello jaeger development team.

We are using your project and we appreciate your work!
Unfortunately, we faced one issue and came to decision to ask you.

The question is _"Why maximum dependency length is hard-coded to 100 in DAG?"_.

```javascript
if (dependencies.length <= 100) {
      GRAPH_TYPE_OPTIONS.push({ type: 'DAG', name: 'DAG' });
}
```

In our project we have a lot of cases with 100+ dependencies and due to this limitation DAG can't be shown.

Thanks in advance.

**Comment by yurishkuro** [comment]
Iirc it's because for large graphs that layout doesn't work well

**Comment by egorzekov** [comment]
Could you please do this setting configurable?

**Comment by tiffon** [comment]
@egorzekov A config setting has been added for this. It's will be released in 1.1.0, which is being prepped:

https://github.com/jaegertracing/jaeger/pull/612

The config setting is `dependencies.dagMaxNumServices`. The UI config docs [are here](http://jaeger.readthedocs.io/en/latest/deployment/#ui-configuration). The specific setting is not added to the docs, yet, but the following config JSON would change the max to 200:

```json
{
  "dependencies": {
    "dagMaxNumServices": 200
  }
}
```
